src/findings/utils.py

```diff
@@ -16,12 +16,13 @@
 
 
 class FindingCategory(enum.Enum):
+    # TODO(xiaozhenliu): renumber the enum values
+    # once they are all verified in use.
     ENUM_VALUE_ADDITION = 1
     ENUM_VALUE_REMOVAL = 2
     ENUM_VALUE_NAME_CHANGE = 3
     ENUM_ADDITION = 4
     ENUM_REMOVAL = 5
-    ENUM_NAME_CHANGE = 6
```

Comment:
enum_name_change is not used anywhere, since the enum is identified by name, enums with different names will not be compared together. Cleanups will be applied once the refactors are done for all comparators.